src/main/java/crazypants/enderio/machine/solar/BlockSolarPanel.java

```diff
@@ -135,25 +138,26 @@ public String getUnlocalizedNameForTooltip(ItemStack itemStack) {
     return getUnlocalizedName();
   }
 
-  @Override
-  public void getWailaInfo(List<String> tooltip, EntityPlayer player, World world, int x, int y, int z) {
-    TileEntity te = getTileEntity(world, new BlockPos(x, y, z));
-    if(te instanceof TileEntitySolarPanel) {
-      TileEntitySolarPanel solar = (TileEntitySolarPanel) te;
-      float efficiency = solar.calculateLightRatio();
-      if(!solar.canSeeSun()) {
-        tooltip.add(TextFormatting.RED + EnderIO.lang.localize("tooltip.sunlightBlocked"));
-      } else {
-        tooltip.add(String.format("%s : %s%.0f%%", TextFormatting.WHITE + EnderIO.lang.localize("tooltip.efficiency") + TextFormatting.RESET,
-            TextFormatting.WHITE, efficiency * 100));
-      }
-    }
-  }
-
-  @Override
-  public int getDefaultDisplayMask(World world, int x, int y, int z) {
-    return 0;
-  }
+//  @Override
```

Comment:
FYI: I completely dropped WAILA integration in favor of TOP. It was so out of date that it'll be better to re-implement it using the TOP data collector if we ever want it back.